src/printer.js

```diff
@@ -1,4 +1,26 @@
 // https://prettier.io/docs/en/plugins.html#printers
-const genericPrint = (path, options) => options.originalText;
+const { concat, hardline, join } = require('prettier').doc.builders;
+
+const space = ' ';
```

Comment:
is there a better way to deal w/ spaces?

Reply:
Generally you’re going to want to use `line` and `indent()` if the language will allow you to insert a newline at that specific place. In the cases (rare for JS, perhaps not so much for other languages) where you can’t break, using a literal `' '` can help make your intentions clear.